docs/Configuration.md

```diff
@@ -205,6 +207,13 @@ simple id, for everything above hash_id will be forced). Default value:
 
 Integer working as a label to associate a test to a specific Test Agent.
 
+### age_reuse_previous_test
+
+Positiv integer (in seconds) for how old a previous test of the same
+zone name and parameters must be before we start a new test. Internally
+the value is converted to whole minutes. If the conversion results in
+zero minutes, then the default value (600 seconds) is used.
```

Comment:
Why is the unit seconds here, rather than minutes?

Reply:
I felt that seconds was a better unit, especially since other keys take seconds. I would rather turn it around and make it seconds all the way through. In the MySQL adapter it is converted to seconds before use.

If this is merged I will create a second PR where the time unit is seconds all the way through.

But I will not insist. If minutes is better I can take that.

Reply:
I see. Uniform units sure is a nice feature. Seconds it is. If we could also get rid of the rounding, that would be great!